### PR TITLE
Manually optimized folder-sync

### PR DESCRIPTION
**Pull request by jguddas**
## What is the purpose of this pull request?

- [ ] New Icon
- [ ] Bug fix
- [ ] New Feature
- [ ] Documentation update
- [x] Other: Minified path

## Before Submitting 

- [x] I've read the [Contribution Guidelines](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).
- [x] I've checked if there was an existing PR that solves the same issue.

### CODE REVIEW

File: icons/folder-sync.svg [hunk 1]
```diff
@@ -11,7 +11,7 @@
 >
   <path d="M9 20H4a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h3.9a2 2 0 0 1 1.69.9l.81 1.2a2 2 0 0 0 1.67.9H20a2 2 0 0 1 2 2v1" />
   <path d="M12 10v4h4" />
-  <path d="m12 14 1.5-1.5c.9-.9 2.2-1.5 3.5-1.5s2.6.6 3.5 1.5c.4.4.8 1 1 1.5" />
+  <path d="m12 14 1.5-1.5a4.95 4.95 0 0 1 8 1.5" />
   <path d="M22 22v-4h-4" />
-  <path d="m22 18-1.5 1.5c-.9.9-2.1 1.5-3.5 1.5s-2.6-.6-3.5-1.5c-.4-.4-.8-1-1-1.5" />
+  <path d="m22 18-1.5 1.5a4.95 4.95 0 0 1-8-1.5" />
```
Comment (karsa-mistmere) [inline icons/folder-sync.svg:16]: This optimisation (once again) takes the lines too much off grid in my opinion.
The original lines were a very neat 10×10px in size, this brings them to 10×9.86.

**Review by karsa-mistmere — APPROVED** [review]

File: icons/folder-sync.json [hunk 2]
```diff
@@ -1,7 +1,8 @@
 {
   "$schema": "../icon.schema.json",
   "contributors": [
-    "danielbayley"
+    "danielbayley",
+    "jguddas"
```
Comment (ericfennis) [inline icons/folder-sync.json:5]: Not sure if you made this one or is automatically added. Is this change intentional?

Comment (jguddas) [reply icons/folder-sync.json:5]: It's a manuall edit, not an automatic optimization using https://lucide-studio.vercel.app.

Comment (ericfennis) [reply icons/folder-sync.json:5]: True but the design is not changed right?

Comment (jguddas) [reply icons/folder-sync.json:5]: Not significantly.

image

**Review by ericfennis — APPROVED** [review]
